test/decimal_test.exs

```diff
@@ -500,7 +500,7 @@ defmodule DecimalTest do
     assert Decimal.reduce(~d"-2.00") == d(-1, 2, 0)
     assert Decimal.reduce(~d"200") == d(1, 2, 2)
     assert Decimal.reduce(~d"0") == d(1, 0, 0)
-    assert Decimal.reduce(~d"-0") == d(-1, 0, 0)
+    assert Decimal.reduce(~d"-0") == d(1, 0, 0)
```

Comment:
@IcyEagle I'm not sure about this change of behavior; the contract is (emphasis mine):

>    Reduces the given number. Removes trailing zeros from coefficient while keeping the **number numerically** equivalent by increasing the exponent.

and worth noting that:

```elixir
iex(16)> Decimal.equal?(Decimal.new("1"), Decimal.new("-1"))
false
```

Reply:
> by increasing the exponent
Increasing of exponent for zero doesn't change its absolute value :)

Zero is the only special value among all values which in fact should have no sign since it has no sense. Since Decimal has `sign` field I would recommend to cast it to one of two available "shapes" (positive or negative) to eliminate confusion.